### PR TITLE
Label sync v2

### PR DESCRIPTION
**Pull request by Johennes**

### CODE REVIEW

**Comment by Johennes** [comment]
@t3chguy I'm not sure how practical it is to review but maybe you could give it a sanity check before I start testing it in the wild?

File: .github/actions/sync-labels.ts
Comment (t3chguy) [inline .github/actions/sync-labels.ts:1]: copyright :D?

**Review by t3chguy — APPROVED** [review]
trial by fire?

File: .github/actions/sync-labels.ts
Comment (Johennes) [reply .github/actions/sync-labels.ts:1]: Good call!

**Pull request merged**